src/emu/render.cpp

```diff
@@ -1620,6 +1629,28 @@ void render_target::load_layout_files(const internal_layout *layoutfile, bool si
 			throw emu_fatalerror("Couldn't parse default layout??");
 	}
 
+	// Check the parent of the parent to cover bios based artwork
+	if (cloneof != -1) {
+		const game_driver &clone(driver_list::driver(cloneof));
+		int cloneofclone = driver_list::clone(clone);
+		if (cloneofclone != -1 && cloneofclone != cloneof)
+		{
+			if (!load_layout_file(driver_list::driver(cloneofclone).name, driver_list::driver(cloneofclone).name))
+				have_default |= load_layout_file(driver_list::driver(cloneofclone).name, "default");
+			else
+				have_default |= true;
+		}
+	}
+
+	// Use fallback artwork if defined and no artwork has been found yet
+	if (m_manager.machine().options().fallback_artwork())
```

Comment:
You probably don't want to load this if machine-specific artwork was found - the name suggests a fallback, not additional views for everything.

Reply:
Question: should fallback artwork be loaded if:
- parent artwork is found?
- bios artwork is found?
- internal layout is found?